policy/modules/kernel/files.if

```diff
@@ -736,6 +736,24 @@ interface(`files_read_all_files',`
 	')
 ')
 
+########################################
+## <summary>
+##	Map all files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`files_mmap_all_files',`
+	gen_require(`
+		attribute file_type;
+	')
+
+	allow $1 file_type:file map;
```

Comment:
I'd prefer to change this to an `mmap_read` interface.

Reply:
Summary also needs an update.

Reply:
What is funny is I started with 'files_mmap_read_all_files' interface, but then went with this.  Oh well.  I did have to add the 'optional' block for shadow files otherwise there is a complaint at compile time.  Let me know if you want any other changes.

Reply:
Updated summary also.